.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx pretty-quick --staged
+npm run lint
```

Comment:
It might be worth switching from `pretty-quick` to [`lint-staged`](https://github.com/okonet/lint-staged). This implementation here will run lint on the entire codebase on every commit, rather than just changed files.

However, since a full lint on this repo takes less than a second (on my laptop), this is probably fine for now and we can iterate on this later.